builder/steps/gen-dockerfile/contents/test/detect_setup_test.ts

```diff
@@ -85,6 +85,20 @@ function eachOf(expectedLogs: string[]): StringArrayVerifier {
   };
 }
 
+function noneOf(givenLogs: string[]): StringArrayVerifier {
+  return (acutalLogs: string[]) => {
+    const notExpected = new Set(givenLogs);
+    const found = new Set();
```

Comment:
Nit. Maybe something like this is simpler than building the found set and comparing later?

`assert(actualLogs.every(actual => !notExpected.has(actual)));`

Reply:
That is simpler, but I was thinking that if the test failed, then the way I wrote it would allow the printed failure to indicate which items were found that caused the failure.

Reply:
Makes sense.